spec/notifications/new_aircraft_created_notification_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe NewAircraftCreatedNotification, type: :model do
+
+  context 'when aircraft are created' do
+    it 'creates notifications' do
+
+      # Must eagerly create a user first, or notifications will not be created
+      user = FactoryBot.create(:user)
+      aircraft = FactoryBot.create(:aircraft)
+
+      # TODO: instead of count, very actual contents of notifications
+      # Note also, notifications temporarily set to User.all
+
+      expect(aircraft).to be_valid
+      expect(user).to be_valid
+      expect(Notification.count).to be(3)
+
+      FactoryBot.create(:aircraft)
+      expect(Notification.count).to be(4)
+    end
+
+    it 'notifies the correct user' do
+      # Note, notification is temporarily User.All
+      FactoryBot.create(:user, first_name: "Sally")
+      FactoryBot.create(:aircraft)
+      expect(Notification.last.recipient.first_name).to eq("Sally")
```

Comment:
perhaps instead of checking for the `first_name` we can check if the notification was properly delivered to the `user`.
i.e. `expect(Notification.last.recipient).to eq(user)`